hack/lib/etcd.sh

```diff
@@ -68,8 +68,17 @@ kube::etcd::cleanup() {
 kube::etcd::install() {
   (
     cd "${KUBE_ROOT}/third_party"
-    curl -fsSL --retry 3 https://github.com/coreos/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz | tar xzf -
-    ln -fns "etcd-v${ETCD_VERSION}-linux-amd64" etcd
+    uname=$(uname)
+    if [[ "${uname}" == "Darwin" ]]; then
```

Comment:
other minor nit: probably don't need uname as a variable. just `if [[ $(uname) == "Darwin" ]]` would be fine.